### DISCUSSION TITLE
Is it possible to specify the type of each column in my tabular data instead of letting gluon detect it?

### DISCUSSION THREAD

**Discussion by TuanNguyen27** [autogluon/autogluon#2511] [General]
The aim is to have more control over the feature engineering that's provided by gluon. If yes, is there a tutorial on how to do this ?

**Comment by gradientsky** [comment]
You can define your own feature engineering pipelines. Please see [Feature Engineering tutorial](https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-feature-engineering.html#customization-of-feature-engineering) and [in-depth examples](https://github.com/autogluon/autogluon/blob/master/examples/tabular/example_custom_feature_generator.py).
